app/channel_test.go

```diff
@@ -103,78 +103,121 @@ func TestRemoveAllDeactivatedMembersFromChannel(t *testing.T) {
 }
 
 func TestMoveChannel(t *testing.T) {
-	th := Setup(t).InitBasic()
-	defer th.TearDown()
-	var err *model.AppError
+	t.Run("should move channels between teams", func(t *testing.T) {
```

Comment:
Please make a little refactor and create a different t.Run scenarios where:

- Move a channel without problem
- Move a channel fails because of member mismatch
- Move a channel with deactivated users fails
- Move a channel with deactivated users with the option works
- Move a channel with no members

This way is cleaner and readable when you're checking the tests

Reply:
It would be nicer, but I don't have the time to do that now. This PR is just focused on adding the new test to confirm that this works with the sidebar correctly, and moving the rest of this into a subtest was just a side effect of that.

Reply:
Ok, but is not just nicer is something really desirable. For example, when I'm working on something new, I tend to read the tests to know how it works so treating them as any other production code is a good practice. My suggestion was just cut that code and create differen test cases